service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageHearingsControllerHearingInPastMidEventTest.java

```diff
@@ -0,0 +1,142 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.OverrideAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.service.FeatureToggleService;
+import uk.gov.hmcts.reform.fpl.service.ManageHearingsService;
+import uk.gov.hmcts.reform.fpl.service.time.Time;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+import static java.lang.Long.parseLong;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+@ActiveProfiles("integration-test")
+@OverrideAutoConfiguration(enabled = true)
+@WebMvcTest(ManageHearingsController.class)
+public class ManageHearingsControllerHearingInPastMidEventTest extends AbstractControllerTest {
+
+    private static final String CASE_ID = "12345";
+
+    @MockBean
+    private FeatureToggleService featureToggleService;
+
+    @MockBean
+    private ManageHearingsService hearingsService;
+
+    @Autowired
+    private Time time;
+
+
+    ManageHearingsControllerHearingInPastMidEventTest() {
+        super("manage-hearings");
+    }
+
+    @Test
+    void shouldPopulateHearingDatesToCorrectOneWhenIncorrectAndToggledOn() {
+        given(featureToggleService.isAddHearingsInPastEnabled()).willReturn(true);
+
+        LocalDateTime correctStartDate = LocalDateTime.of(2050, 6, 5, 11, 0);
+        LocalDateTime correctEndDate = LocalDateTime.of(2050, 6, 5, 11, 0);
```

Comment:
Might be better declaring a NOW localDateTime variable and just using .minusDays(2) etc to make it easier to read.